src/Compatibility/GenAPI/Microsoft.DotNet.GenAPI/GenAPIApp.cs

```diff
@@ -32,58 +30,20 @@ public static void Run(ILog logger,
             bool respectInternals,
             bool includeAssemblyAttributes)
         {
-            bool resolveAssemblyReferences = assemblyReferences?.Length > 0;
```

Comment:
This logic that creates the loader is what I need to share with ApiDiff, so it's been moved up to AssemblyLoaderFactory. So the loader and the dictionary of IAssemblySymbols are now being passed as the first two arguments.